syllabus/4b-dApps_Off-Chain/2-papi-starter-slides.md

```diff
@@ -3,15 +3,88 @@ title: Polkadot-API Getting Started
 description: Getting started with Polkadot-API
 ---
 
-# Polkadot-API
+# dApps / Off-chain
 
-## Getting Started
+---
+
+## Module Overview
+
+- Understand interaction with a chain
+- Metadata in-depth <!-- .element: class="fragment" -->
+- JSON-RPC Interface <!-- .element: class="fragment" -->
+  - chainHead <!-- .element: class="fragment" -->
+  - storage <!-- .element: class="fragment" -->
+  - transactions <!-- .element: class="fragment" -->
+
+---
+
+## Module Overview
+
+- Polkadot-API
+  - Sub-libraries <!-- .element: class="fragment" -->
+  - Subscriptions <!-- .element: class="fragment" -->
+  - Compatibility API <!-- .element: class="fragment" -->
+  - Signers <!-- .element: class="fragment" -->
+  - Ink <!-- .element: class="fragment" -->
+- PAPI SDKs <!-- .element: class="fragment" -->
+
+---
+
+## Grading
+
+- Assignment
+  - dApp for specific chain feature <!-- .element: class="fragment" -->
+  - Optional Add-ons <!-- .element: class="fragment" -->
+  - Qualitative grading <!-- .element: class="fragment" -->
+- UI/UX is NOT graded <!-- .element: class="fragment" -->
+- Business logic code <!-- .element: class="fragment" -->
+- Research, considerations <!-- .element: class="fragment" -->
+
+Notes:
+
+- Not Met Expectations – The student either did not complete the assignment on time or submitted work that does not meet the required standards.
+- Met Expectations – The student successfully completed the assignment, adhering to the provided guidelines, with well-structured and competent code.
+- Exceeded Expectations – The student delivered a high-quality assignment, demonstrating excellent coding practices, incorporating optional challenges, and going beyond the expected outcome.
```

Comment:
We will be grading 0-10 but we must not share that to the student.

Overall we will not be sharing grades with the students... Only feedback on their work and if they are complted, graduated, or graduated with distinction